src/vs/workbench/contrib/search/browser/searchActionsFind.ts

```diff
@@ -347,11 +351,29 @@ function getMultiSelectedSearchResources(viewer: WorkbenchCompressibleObjectTree
 		.filter((renderableMatch): renderableMatch is URI => (renderableMatch !== null));
 }
 
-export function findInFilesCommand(accessor: ServicesAccessor, args: IFindInFilesArgs = {}) {
+export async function findInFilesCommand(accessor: ServicesAccessor, _args: IFindInFilesArgs = {}) {
+
 	const searchConfig = accessor.get(IConfigurationService).getValue<ISearchConfiguration>().search;
+	const viewsService = accessor.get(IViewsService);
+	const args: IFindInFilesArgs = {};
+	if (Object.keys(_args).length !== 0) {
+		const configurationResolverService = accessor.get(IConfigurationResolverService);
+		const historyService = accessor.get(IHistoryService);
+		const workspaceContextService = accessor.get(IWorkspaceContextService);
+		const activeWorkspaceRootUri = historyService.getLastActiveWorkspaceRoot(Schemas.file);
+		const lastActiveWorkspaceRoot = activeWorkspaceRootUri ? withNullAsUndefined(workspaceContextService.getWorkspaceFolder(activeWorkspaceRootUri)) : undefined;
+
+		for (const entry of Object.entries(_args)) {
```

Comment:
this copies what the search editor does here:
https://github.com/microsoft/vscode/blob/8b76efe9d317d50cb5b57a7658e09ce6ebffaf36/src/vs/workbench/contrib/searchEditor/browser/searchEditorActions.ts#L152-L158

Reply:
good point; added a comment that references the source